p2p/switch.go

```diff
@@ -585,9 +585,7 @@ func (sw *Switch) AddUnconditionalPeerIDs(ids []string) error {
 }
 
 func (sw *Switch) isPeerPersistentFn() func(*NetAddress) bool {
```

Comment:
do we need this func at all?

Reply:
I can't see a reason to keep it, but I also can't speak why its there. If I remove it then everything works as intended, so if want to remove we can

Reply:
Let's remove